scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorTask.java

```diff
@@ -506,6 +506,14 @@ public void fail() {
     actor.onActorFailed();
   }
 
+  public int estimateQueueLength() {
+    if (fastLaneJobs instanceof ClosedQueue || submittedJobs instanceof ClosedQueue) {
+      return 0;
+    }
+    // TODO: In theory this could overflow. We could consider using a saturating add using a long
```

Comment:
:x: Here is a todo, are we expecting to fix this soon? Or can we do it right now?